spec/predicates/human-review-crev.md

```diff
@@ -12,21 +12,25 @@ Version: v0.1.0
 
 This attestation type is used to describe the results of human review of
 dependency source code. The format is based on the
-[crev project](https://github.com/crev-dev/crev).
+[crev project](https://github.com/crev-dev/crev), specifically the
+[package review proof](https://github.com/crev-dev/cargo-crev/blob/master/crev-lib/rc/doc/editing-package-review.md)
+defined in the [cargo-crev](https://github.com/crev-dev/cargo-crev)
+implementation.
 
 ## Use Cases
 
-As noted above, crev enables social review of popular open source software
-dependencies. A crev review includes information such as the thoroughness of
-the review, understanding of the source code, and a final rating.
+crev enables social review of popular open source software dependencies. A crev
+review includes information such as the thoroughness of the review,
+understanding of the source code, and a final rating. The ratings for these
```

Comment:
Being very nitpicky here, though I think this conveys the expectations more clearly.
```suggestion
understanding of a package's source code, and a final rating. The ratings for these
```